### PR TITLE
github actions for CI

### PR DESCRIPTION
**Pull request by BenjaminHabert**

### CODE REVIEW

**Comment by agramfort** [comment]
Can you use conda as it is what is used on the server and aws ? So use an environment.yml file

File: environment.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  - tensorflow==2.6.2
+
+  # dev
+  - black
+  - flake8
+
+  - pip
+  - pip:
+    - osfclient
+    - ramp-workflow
```
Comment (agramfort) [inline environment.yml:20]: missing empty line

**Review by agramfort — COMMENTED** [review]
do we have a test to see if requirements.txt is in sync with conda env in other kits?

**Comment by BenjaminHabert** [comment]
> 

yes there is a mechanism to test requirements == environment in the meg repo; will add here.

**Comment by agramfort** [comment]
Thx 🙏 @BenjaminHabert

**Pull request merged**